metadata-ingestion/src/datahub/ingestion/source/data_lake_common/path_spec.py

```diff
@@ -63,6 +63,11 @@ class Config:
         description="Not listing all the files but only taking a handful amount of sample file to infer the schema. File count and file size calculation will be disabled. This can affect performance significantly if enabled",
     )
 
+    allow_double_stars: bool = Field(
```

Comment:
is double star supported on all sources that use path spec? do we have tests on the data lake source that test the ** behavior for includes?

I know it always works in the exclude paths, but not include iirc

Reply:
Double star is a valid glob pattern and whichever source support glob, it should support double star as well.